ejabberd_bridge/management/commands/ejabberd_auth.py

```diff
@@ -48,11 +49,11 @@ def to_ejabberd(self, answer=False):
         sys.stdout.write(b.decode("utf-8"))
         sys.stdout.flush()
 
-    def auth(self, username=None, server="localhost", password=None):
+    def auth(self, user_id=None, server="localhost", password=None):
         self.logger.debug("Authenticating %s on server %s" % (username, server))
 
         try:
-            user_obj = get_user_model().objects.get(id=username)            
+            user_obj = user_model.objects.get(id=user_id)            
```

Comment:
`user_model` here need to be called with `self` or class name `Command`. alternatively, you can put the user_model initialization globally (outside of the class)